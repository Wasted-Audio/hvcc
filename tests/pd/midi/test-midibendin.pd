#N canvas 813 354 503 302 12;
#X obj 97 132 print;
#X obj 97 61 bendin;
#X obj 97 96 pack f f;
#X connect 1 0 2 0;
#X connect 1 1 2 1;
#X connect 2 0 0 0;
