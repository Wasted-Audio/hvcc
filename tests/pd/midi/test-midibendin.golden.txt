[@ 0.000] print: 0 1
[@ 0.000] print: 8192 2
[@ 0.000] print: 1 1
[@ 0.000] print: 8193 2
[@ 0.000] print: 2 1
[@ 0.000] print: 8194 2
[@ 0.000] print: 3 1
[@ 0.000] print: 8195 2
[@ 0.000] print: 4 1
[@ 0.000] print: 8196 2
[@ 0.000] print: 5 1
[@ 0.000] print: 8197 2
[@ 0.000] print: 6 1
[@ 0.000] print: 8198 2
[@ 0.000] print: 7 1
[@ 0.000] print: 8199 2
[@ 0.000] print: 8 1
[@ 0.000] print: 8200 2
[@ 0.000] print: 9 1